mern-ecommerce/routes/auth.js

```diff
@@ -12,5 +12,6 @@ const { userSignupValidator } = require('../validator');
 router.post('/signup', userSignupValidator, signup);
 router.post('/signin', signin);
 router.get('/signout', signout);
-
+router.post('/forgot-password', forgotPassword);
+router.put('/resetPassword', resetPassword);
```

Comment:
The function `resetPassword` is not imported or defined in the provided file context. It is being used in the router configuration without being defined or imported. This will result in a runtime error.